docs/markdown/Builtin-options.md

```diff
@@ -144,3 +144,9 @@ shell32, ole32, oleaut32, uuid, comdlg32, advapi32.
 c_args, cpp_args, c_link_args, and cpp_link_args only affect native builds,
 when cross compiling they will not be applied to binaries or libraries
 targeting the host system, only those being run on the build system.
+
+When using MSVC, `cpp_eh=default` will result in `/EHsc`, with the other values
+causing meson to pass `/EH[value]`, or nothing at all in the case of
+`cpp_eh=none`. When using other compilers, `cpp_eh=default` does nothing
+(allowing the default, exceptions, to work), while `cpp_eh=none` passes
+`-fno-exceptions`.
```

Comment:
This should mention that this behaviour is only available starting with release 0.51.0.

Reply:
I'll do that later, currently have class.